src/MakeListingDisplay/components/MakeListingDisplay.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+
+import InputField from './InputField';
+
+import '../styles/MakeListingDisplay.css';
+
+/**Component class that displays the input fields for a user to post a listing */
+class MakeListingDisplay extends Component {
+    render(){
+        return (
+            <div>
+                <div id="header">
+                    <h1>Post a Listing</h1>
+                </div>
+                <form action="/listings" id="listingForm" method="POST">
+                    <InputField fieldHeader="Title:" fieldName="title" fieldType="text" />
+                    <InputField fieldHeader="Description:" fieldName="description" fieldType="text" />
+                    <InputField fieldHeader="Total number of rooms in apartment:" fieldName="numRooms" fieldType="number" />
+                    
+                    <p>Number of rooms for rent:</p>
+                    <div id="singleInfo">
+                        <InputField fieldHeader="Singles:" fieldName="numSingles" fieldType="number" />
+                        <InputField fieldHeader="Monthly Rent Per Single:" fieldName="singlePrice" fieldType="number" />
+                    </div>
```

Comment:
It would be nice if we could add some client side validation to these fields. For price, a step=0.01 would make sense. I think making all the fields required would also make sense, as well as setting all of the numbers to have a minimum of 0

Reply:
I implemented the step and min for price and number inputs in InputField.jsx